addons/l10n_es_pos/models/pos_config.py

```diff
@@ -4,24 +4,30 @@
 class PosConfig(models.Model):
     _inherit = "pos.config"
 
-    is_spanish = fields.Boolean(string="Company located in Spain", compute="_is_company_spanish")
-
-    def _is_company_spanish(self):
-        for pos in self:
-            pos.is_spanish = pos.company_id.country_id.code == "ES"
-
+    is_spanish = fields.Boolean(string="Company located in Spain", compute="_compute_is_spanish")
     l10n_es_simplified_invoice_limit = fields.Float(
         string="Simplified Invoice limit amount",
         help="Over this amount is not legally possible to create a simplified invoice",
         default=400,
     )
-    l10n_es_simplified_invoice_journal_id = fields.Many2one('account.journal')
+    l10n_es_simplified_invoice_journal_id = fields.Many2one(
+        comodel_name='account.journal',
+        domain="[('type', '=', 'sale')]",
+        check_company=True,
+        help="Only if this journal is set, simplified invoices will be created for every not already invoiced PoS order.",
+    )
     simplified_partner_id = fields.Many2one(
```

Comment:
check_company=True?

Reply:
Is it relevant for a non-stored computed field ? :thinking:

Reply:
Hmm, not really.